### PR TITLE
7: Added github action for phpunit

### LINKED ISSUE #7: Automated testing
Automated testing with github actions.

### PR DESCRIPTION
**Pull request by noahlvb**
resolves #7

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+    unit-tests:
+        name: Unit tests
+        runs-on: ubuntu-latest
```
Comment (blackshadev) [inline .github/workflows/ci.yml:8]: Does it needs to be ubuntu? Isn't alpine faster / smaller?

**Review by blackshadev — APPROVED** [review]
Question only, you can consider it or merge it. Great work!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (noahlvb) [reply .github/workflows/ci.yml:8]: Sorry not possible I'm disappointed as well.

image

Comment (noahlvb) [reply .github/workflows/ci.yml:8]: And I cannot merge anything cause I don't have write access on this repo 😄

Comment (blackshadev) [reply .github/workflows/ci.yml:8]: Merged and I will change that.

**Pull request merged**